src/api/l_audio.c

```diff
@@ -24,8 +24,9 @@ static int l_lovrAudioReset(lua_State* L) {
 
 static int l_lovrAudioStart(lua_State* L) {
   AudioType type = luax_checkenum(L, 1, AudioType, "playback");
-  lovrAudioStart(type);
-  return 0;
+  bool ret = lovrAudioStart(type);
+  lua_pushboolean(L, ret);
```

Comment:
I'd prefer a name with a little more meaning like `success` / `started` or something (or just to have it inlined into the pushboolean call to avoid naming altogether).